matplotlibcpp.h

```diff
@@ -438,6 +447,61 @@ bool hist(const std::vector<Numeric>& y, long bins=10,std::string color="b", dou
     return res;
 }
 
+template< typename Numeric>
+bool bar(const std::vector<Numeric>& y, const std::map<std::string, std::string>& keywords = {})
+{
+    PyObject* yarray = get_array(y);
+
+    std::vector<int> x;
+    for (int i = 0; i < y.size(); i++)
+        x.push_back(i);
+
+    PyObject* xarray = get_array(x);
+
+    PyObject* kwargs = PyDict_New();
+
+    PyDict_SetItemString(kwargs, "ec", PyString_FromString("black"));
```

Comment:
I think since you're already taking a `keywords` argument to the c++ function, the keys in that should override the default values specified here (and the rest passed through to the python function)